DevoxxRemoteFunctions/build.gradle

```diff
@@ -1,4 +1,26 @@
+import jp.classmethod.aws.gradle.lambda.AWSLambdaUpdateFunctionCodeTask
+
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath "jp.classmethod.aws:gradle-aws-plugin:0.+"
```

Comment:
I prefer to depend on a specific version, especially if it's still in the 0.x phase. This might cause unexpected issues at a later point in time when the plugin is updated with backward incompatible changes.